README.md

```diff
@@ -98,6 +97,9 @@ pong
 (32ms)
 ```
 
+* There can only be a single handler for an endpoint path. The handler is expected to dispatch by HTTP method (verb).
+* The handler lambda syntax in the REPL mimics C++11 to a certain extent -- no full lambda support guaranteed.
```

Comment:
Nits: `to certain extent` (no "a"), and `no full support is guaranteed` (insert "is").

It's a great and well-designed open-source project. Let's keep the doc clean as well :)